### DISCUSSION TITLE
ver 9.5 ruined all

### DISCUSSION THREAD

**Discussion by alexei-moneyr** [postmanlabs/postman-flows#169] [Defect]
Guys why?! What did you done in latest update?
Variables deprecated, durables?! what they are? Now if i connect durable to request I need to manually enter all fields (variables did it best) 
 How can I concentrate more than two objects?

**Comment by saswatds** [comment]
@alexei-moneyr Could you explain to us a bit about what you are trying to achieve? And what went wrong. 

P.S - `Durables` is not a replacement for variables. We deprecated variables because we added the ability to set variables within the `Send Request` block itself. We are in the process of building https://github.com/postmanlabs/postman-flows/discussions/133 which will let you create complex data structures that variables cannot support because of its table like nature. If your use-case involved merging data from two different sources please feel free to continue using `Create Variables` and `Assign Variables`

**Reply by samay-v** [reply]
@alexei-moneyr I've tried to explain the use of Durables in [this post](https://github.com/postmanlabs/postman-flows/discussions/171) with an example video. It would probably help you understand it better 
https://github.com/postmanlabs/postman-flows/discussions/171
